root/etc/nginx/nginx.conf

```diff
@@ -60,65 +60,17 @@ http {
           etag off;
           if_modified_since off;
           more_clear_headers 'Last-Modified';
+          location =/index.html {
+            expires epoch;
```

Comment:
epoch will set cache expires to very long past (year 1970) which in effect works like expire always.